crypto/bls/benches/benchmark.rs

```diff
@@ -0,0 +1,53 @@
+use std::borrow::Cow;
+
+use bls::{verify_signature_sets, Hash256, PublicKey, SecretKey, SignatureSet};
+use criterion::{criterion_group, criterion_main, Criterion};
+
+pub fn bench_verify_signature_set(c: &mut Criterion) {
+    let private_keys: Vec<_> = (0..16384).map(|_| SecretKey::random()).collect();
+    let public_keys: Vec<Cow<PublicKey>> = private_keys
+        .iter()
```

Comment:
added benchmarks to determine if this change is useful in any particular context